physics/module_sf_ruclsm.F90

```diff
@@ -19,28 +19,56 @@ MODULE module_sf_ruclsm
    public :: lsmruc, ruclsminit, rslf
 
 !> CONSTANT PARAMETERS
-      real (kind=kind_phys), parameter :: P1000mb = 100000.
-      real (kind=kind_phys), parameter :: xls     = 2.85E6
-      real (kind=kind_phys), parameter :: rhowater= 1000.
-      real (kind=kind_phys), parameter :: piconst = 3.1415926535897931
-      real (kind=kind_phys), parameter :: r_v     = 4.6150e+2
+!! @{
+      real (kind_phys), parameter :: P1000mb = 100000._kind_dbl_prec
+      real (kind_phys), parameter :: xls     = 2.85E6_kind_dbl_prec
+      real (kind_phys), parameter :: rhowater= 1000._kind_dbl_prec
+      real (kind_phys), parameter :: piconst = 3.1415926535897931_kind_dbl_prec
```

Comment:
This is defined as `pi`.

Reply:
Now the code uses constants from Physcons.